src/components/CodeEditorWindow/config/RequesFromCode.js

```diff
@@ -30,11 +48,19 @@ export function codeParse(codeText) {
     try {
       reqBody = body === "\n" ? {} : JSON.parse(body);
     } catch (e) {
-      return { method: null, endpoint: null, reqBody: null, error: "Fix the Position brackets to run & check the json" }
+      return { method: null, endpoint: null, reqBody: body, error: "Fix the Position brackets to run & check the json" }
     }
   }
-  if (method === "") {
-    return { method: null, endpoint: null, reqBody: null, error: "Add headline or remove the line gap between json and headline (if any)" }
+  if (method === "" && endpoint === "") {
+    return { method: null, endpoint: null, reqBody: reqBody, error: "Add Headline or remove the line gap between json and headline (if any)" }
+  }
+  else if (method === "") {
+    return { method: null, endpoint: endpoint, reqBody: reqBody, error: "Add method" }
   }
+  else if (endpoint === "") {
+    return { method: method, endpoint: null, reqBody: reqBody, error: "Add endpoint" }
+  }
+  else{
   return { method: method, endpoint: endpoint, reqBody: reqBody, error: null }
+  }
```

Comment:
pls use code formatting. (Might be worth introducing it as a pre-commit hook later)